src/components/Thought.js

```diff
@@ -459,6 +468,17 @@ const ThoughtContainer = ({
   const styleContainer = getStyle(state, thoughts, { container: true })
   const styleContainerZoom = isEditingPath ? getStyle(state, thoughts.concat('=focus', 'Zoom'), { container: true }) : null
 
+  const cursorOnAlphabeticalSort = cursor && attributeEquals(state, context, '=sort', 'Alphabetical')
+
+  const draggingThoughtPath = pathToContext(state.draggingThought)
+  const draggingThoughtValue = draggingThoughtPath && draggingThoughtPath[draggingThoughtPath.length - 1]
```

Comment:
Use `contextOf` instead of indexing into `draggingThoughtPath` manually.

Reply:
@raineorshine How can I use `contextOf` in this case? It returns all elements of array except the last but I need only the last.

Reply:
Sorry, I meant `head`!